src/js/site/expandImage.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Add enlarge image capability for easier readability
+ */
+export function expandImageOnClick() {
+  // create references to the modal...
+  const modal = document.getElementById('expandModal');
+
+  // get all images without class icon or logo, inside or article but also have src `.png`
+  // it is most likely that a icon or logo is src `.svg`
+  const images = document.querySelectorAll(
+    "article img:not(a):not(.icon):not(.logo)[src*='.png']",
```

Comment:
since you are avoiding the `icon` class, are all set on the content side to make sure that all the icons have an icon class already?

Reply:
So the main deciding factor is that its .png for none icon and .svg for icon. The icon class and logo class is because I have found an icon that was a png but it is rare. So it gives the extra check.